### DISCUSSION TITLE
Ungrouping Separate Meshes from Imported Models

### DISCUSSION THREAD

**Discussion by dbkdavid** [icosa-gallery/open-brush#605] [General]
Say I have a single OBJ or FBX that has multiple meshes within it, is there a way in Open Brush to ungroup those meshes so that I can manipulate the pieces individually?

I am trying to solve a UI issue here. I am finding it tedious to import large numbers of images from the UI in VR, so I am thinking it would be great if I could embed all my images into a single FBX and then import that into Open Brush as a single model file. This way, I could ungroup the pieces and have a “kit” of meshes to do quick layouts.

**Comment by andybak** [comment]
A couple of things:

1. As a general feature request this is a good idea and it's been something I've been mulling over (it would be for gltf/glb though - I'm not planning any more enhancements to fbx import)
2. Your particular use-case might be easier to solve with scripting. Are you on our Discord? If so - come and chat to me.

Or else - give a bit more info here. What is the original source of the images? Figma or similar? I'm wondering if there's a more direct route directly from another native format or API.

**Reply by dbkdavid** [reply]
Thanks @andybak, sounds like scripting might be the easiest solution. I will get in touch on Discord.

**Comment by mikeskydev** [comment]
Realised in https://github.com/icosa-foundation/open-brush/pull/621

**Comment by andybak** [comment]
Actually - that PR is superseded by #573
